docs/en/enterprise-edition/policy-reference/sast-policies/python-policies/sast-policy-187.adoc

```diff
@@ -0,0 +1,57 @@
+
+== Redis Without SSL
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|=== 
+|Prisma Cloud Policy ID 
+| TBD
+
+|Checkov ID 
+|CKV3_SAST_187
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Language
+|python
+
+|CWEs
+|https://cwe.mitre.org/data/definitions/295.html[CWE-295: Improper Certificate Validation]
+
+|OWASP Categories
+|https://owasp.org/Top10/A02_2021-Cryptographic_Failures/[A02:2021-Cryptographic Failures]
+
+|=== 
+
+
+=== Description
+
+This policy detects if a Redis client is configured to communicate without transport security in a Python application. Deploying such a client in production may leak sensitive information since data transferred are not encrypted. Specifically, two scenarios are covered: (1) when `redis.Redis` is used without setting `ssl` to `True`, and (2) when `redis.Redis.from_url` is used with a `redis://` url.
```

Comment:
This policy detects whether a Redis client in a Python application is configured to communicate without transport security (SSL/TLS). Deploying such a client in production may potentially leak sensitive information as transferred data is not encrypted. Specifically, the following two scenarios are covered: 
- `redis.Redis` is used without setting `ssl` to `True`
- `redis.Redis.from_url` is used with a `redis://` url.